scripts/run_on_popular_crates/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
```

Comment:
I don't think we want to put an entire crate in `scripts`. Future maintainers of this project would find that very confusing, and it's not in line with how Rust projects tend to be structured. Consider moving the crate into its own top-level directory.

Reply:
Can you suggest a good name for the crate?

Reply:
Maybe `check-semver-of-registry-crates`?

Reply:
Perhaps `registry-semver-scanner`? Or, a different direction, `semver-crater` as a riff on `crater`, the tool that tests new Rust releases against all registry crates?

`check-semver-of-registry-crates` is a bit long, especially for something we may need to type in.

Also, depending on the discussion in #86, you may be able to move this to `src/bin/` and configure the `Cargo.toml` to gate this code behind a new `unstable` feature.